### ISSUE TITLE
app.getUserLocale is not a function

### ISSUE THREAD

**Issue by davidmarinangeli** [manekinekko/actions-on-google-i18n#4]
Hello, I'm trying to use your library but as soon as I try to run it, I got this error:

`TypeError: app.getUserLocale is not a function
    at I18n.use (D:\GitHub\Coffee-Advisor\node_modules\@manekinekko\actions-on-google-i18n\index.js:50:22)
    at Object.<anonymous> (D:\GitHub\Coffee-Advisor\functions\index.js:36:6)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at C:\Users\david\AppData\Roaming\npm\node_modules\firebase-tools\lib\triggerParser.js:21:11`

That's how i wrote my webhook with V2 library:

```const {dialogflow, _...here are other AoG components..._} = require('actions-on-google');
const functions = require('firebase-functions');
const i18n = require('@manekinekko/actions-on-google-i18n');

const app = dialogflow();
i18n.use(app);

...

exports.myapp= functions.https.onRequest(app);
```

**Comment by manekinekko** [comment]
Hi, i'm planning to release a new version of this library that is compatible with the dialogflow v2 library. Stay tunned.

**Comment by manekinekko** [comment]
Hi, the v2 has been released under `@sfeir/actions-on-google-i18n@2.0.1` and https://github.com/sfeir-open-source/actions-on-google-i18n/releases/tag/v2.0.1
